src/plugins/Makefile.am

```diff
@@ -221,7 +223,31 @@ libgs_plugin_flatpak_la_SOURCES =			\
 libgs_plugin_flatpak_la_LIBADD = $(GS_PLUGIN_LIBS) $(FLATPAK_LIBS)
 libgs_plugin_flatpak_la_LDFLAGS = -module -avoid-version
 libgs_plugin_flatpak_la_CFLAGS = $(GS_PLUGIN_CFLAGS) $(WARN_CFLAGS)
-endif
+
+libgs_plugin_legacy_external_apps_la_SOURCES =	\
+	gs-appstream.c				\
+	gs-appstream.h				\
+	gs-flatpak.c				\
+	gs-flatpak.h				\
+	gs-flatpak-symlinks.c			\
+	gs-flatpak-symlinks.h			\
+	gs-plugin-legacy-external-apps.c
+libgs_plugin_legacy_external_apps_la_LIBADD = $(GS_PLUGIN_LIBS) $(FLATPAK_LIBS) $(JSON_GLIB_LIBS)
+libgs_plugin_legacy_external_apps_la_LDFLAGS = -module -avoid-version
+libgs_plugin_legacy_external_apps_la_CFLAGS = $(GS_PLUGIN_CFLAGS) $(JSON_GLIB_CFLAGS) $(WARN_CFLAGS)
+
+libgs_plugin_external_runtimes_cleaner_la_SOURCES =		\
```

Comment:
I pondered that and I decided to not include it because the name would be too long and also external runtimes is a term only used on our legacy external apps (i.e. making _legacy_ redundant). That with _cleaner_ seemed enough for me to identify the plugin as something not to keep forever (there's also a short description in the license comment). If you still prefer it, I can of course add it.